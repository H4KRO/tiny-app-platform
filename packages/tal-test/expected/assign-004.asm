main()
  entry:
    Local             name: index
    Local             name: toto
    Attribute         name: property
    Literal "value"
    SetIndex          forceRender: true
