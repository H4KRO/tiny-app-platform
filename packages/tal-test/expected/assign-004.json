[
  {
    "kind": "Assign",
    "address": {
      "kind": "Index",
      "index": {
        "kind": "Local",
        "name": "index"
      },
      "value": {
        "kind": "Attribute",
        "key": "property",
        "value": {
          "kind": "Local",
          "name": "toto"
        }
      }
    },
    "value": "value"
  }
]